xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-notifiers/xwiki-platform-notifications-notifiers-default/pom.xml

```diff
@@ -31,7 +31,7 @@
   <name>XWiki Platform - Notifications - Notifiers - Default bridge</name>
   <description>Default bridge to oldcore for the notifications notifiers API module.</description>
   <properties>
-    <xwiki.jacoco.instructionRatio>0.35</xwiki.jacoco.instructionRatio>
+    <xwiki.jacoco.instructionRatio>0.32</xwiki.jacoco.instructionRatio>
```

Comment:
Out of curiosity, what's causing this decreased ratio?

Reply:
The removal of quite a lot of covered code.